core/src/main/resources/org/apache/spark/ui/static/stagepage.js

```diff
@@ -100,10 +100,10 @@ function getColumnNameForTaskMetricSummary(columnKey) {
             return "Scheduler Delay";
 
         case "diskBytesSpilled":
-            return "Shuffle spill (disk)";
+            return "Spill (disk)";
 
         case "memoryBytesSpilled":
-            return "Shuffle spill (memory)";
+            return "Spill (memory)";
```

Comment:
not related to this change, but what does spill to memory means?

Reply:
`memoryBytesSpilled`  means spill from memory (wasted space in pages will be counted as memory spilled),  and  `diskBytesSpilled` means  spill to disk.
In general, I think `memoryBytesSpilled`  is greater than  `diskBytesSpilled`.

Reply:
![default](https://user-images.githubusercontent.com/24688163/50879786-20497880-1417-11e9-85ab-ae0086b2c5e7.png)
I think it should also be changed here.

Reply:
"spill from memory" and "spill to disk", what's the difference between them?

Reply:
we can take a look at this comment:
 https://github.com/apache/spark/blob/dbbba80b3cb319b147dcf82a69963eee662e289f/core/src/main/java/org/apache/spark/util/collection/unsafe/sort/UnsafeExternalSorter.java#L224

Reply:
The name is pretty confusing. I do not think our end users can understand `Spill(memory)`. cc @srowen

Reply:
@gatorsmile do you think we should remove this metric from the UI entirely then? Removing "shuffle" seems more correct, but then yes, it's not as clear what is 'spilling'. But naming the several things that spill makes this also long and maybe not much easier to understand.